docs/userguide/networking/default_network/build-bridges.md

```diff
@@ -66,11 +67,11 @@ MASQUERADE  all  --  192.168.5.0/24      0.0.0.0/0
 ```
 
 The result should be that the Docker server starts successfully and is now
-prepared to bind containers to the new bridge.  After pausing to verify the
+prepared to bind containers to the new bridge. After pausing to verify the
 bridge's configuration, try creating a container -- you will see that its IP
 address is in your new IP address range, which Docker will have auto-detected.
 
-You can use the `brctl show` command to see Docker add and remove interfaces
+You can use the `brctl show` command to see Docker adds and removes interfaces
```

Comment:
Agreed with @MHBauer here; it's basically `to see ("somebody") add and remove interfaces`.